src/core/client/ui/components/v2/Divider/Divider.css

```diff
@@ -7,17 +7,26 @@ $divider-border-color: var(--palette-grey-300);
 }
 
 .spacing1 {
-  margin: var(--spacing-1) 0;
+  margin-top: var(--spacing-1);
+  margin-bottom: var(--spacing-1);
 }
 
 .spacing2 {
-  margin: var(--spacing-2) 0;
+  margin-top: var(--spacing-2);
+  margin-bottom: var(--spacing-2);
 }
 
 .spacing3 {
-  margin: var(--spacing-3) 0;
+  margin-top: var(--spacing-3);
+  margin-bottom: var(--spacing-3);
 }
 
 .spacing4 {
-  margin: var(--spacing-4) 0;
+  margin-top: var(--spacing-4);
+  margin-bottom: var(--spacing-4);
+}
+
+.horizontallySpaced {
```

Comment:
Would it be more flexible for future usage to make this a spacing that can be passed in that defaults to no spacing if nothing is passed in? So, like, `horizontalSpacing={2}` and then that providers the left and right spacing of 2, but then if nothing gets passed in for it, it defaults to 0?

Reply:
✅